uptycs/README.md

```diff
@@ -0,0 +1,85 @@
+## Setup
+
+To set up this integration, please [Contact Us][2] for an Uptycs account.
+You'll also need Datadog API keys.
+### Configuration
+
+1. Create a [Datadog API key][3].
+2. Create a Datadog Integration Destination on Uptycs platform with Datadog API keys.
+   1. Go to the Configuration > Destinations.
+   2. Click on New destination.
+   3. Select Webhook destination type.
+   3. Provide a name, the DataDog Log collection Url, the API Key as a header.
+
+      ![my screenshot](https://raw.githubusercontent.com/DataDog/integrations-extras/master/uptycs/images/integration_setup_1.png)
+
+   4. Provide your own custom template for alert or detection in the template field.
+
+      ![my screenshot](https://raw.githubusercontent.com/DataDog/integrations-extras/master/uptycs/images/integration_setup_2.png)
+
+   5. Click **save**.
+3. Now that the destination is set up, let's create forwarding rule for it.
+   1. Go to Configuration > Detection Forwarding Rules > New rule
+   2. Provide a name, description, and choose the relevant criteria for the rule.
+   3. Select the created destination in the Destinations options.
+
+      ![my screenshot](https://raw.githubusercontent.com/DataDog/integrations-extras/master/uptycs/images/integration_setup_3.png)
+
+   4. Select Enable Rule and click **save**.
+4. The Created destination can be used for alert forwarding.
+   1. Go to Configuration > Alert Rules.
+   2. Select an Alert Rule or bulk select several rules.
+   3. Select the created destination in the 'Destinations' options.
+   4. Select options 'Notify on Every Alert', 'Close After Delivery'.
+
+      ![my screenshot](https://raw.githubusercontent.com/DataDog/integrations-extras/master/uptycs/images/integration_setup_4.png)
+
+   5. Click **save**.
+6. Once Uptycs generates an alert or detection, it will be delivered as a DataDog Event.
+
+## Data Collected
+
+Each alert contains the following main components:
+   1. Title: A brief info about the alert.
+   2. Description: The description of the alert.
+   3. Id: Uptycs alert ID.
+   4. Uptycs alert code.
+   5. Alert severity.
+   6. Alert key and value.
+   7. Asset details: Asset id and host name.
+   8. Url: Uptycs url to navigate to the Uptycs platform.
+
+      ![my screenshot](https://raw.githubusercontent.com/DataDog/integrations-extras/master/uptycs/images/data_collected_2.png)
+
+Each detection contains the following main components:
+   1. Title or Name: A brief info about the detection.
+   2. Id: Uptycs detection ID.
+   3. Score: Uptycs calculated score.
+   4. Alerts: List of Alerts associated with the detection.
+   5. Events: List of Events associated with the detection.
+   5. Attack Matrix: Techniques associated with the alerts and events.
+   7. Asset details: Asset id and host name.
+   8. Url: Uptycs url to navigate to the Uptycs platform.
+
+      ![my screenshot](https://raw.githubusercontent.com/DataDog/integrations-extras/master/uptycs/images/data_collected_1.png)
```

Comment:
The Data Collected tab unfortunately does not support custom content - it will only render the metrics sent in by your integration if metrics are being sent in. I'd recommend placing this in the Overview section instead, under it's own header like `### Alert Details`

Reply:
However, note that inline images will not render under the Overview section. Images can either be added inline to the Setup section, or to the media carousel by adding them to the manifest.json file.

Reply:
updated